### ISSUE TITLE
Tailwind recepie not working

### ISSUE THREAD

**Issue by agamm** [blitz-js/blitz#1535]
### What is the problem?
When running `npm run start` after installing the tailwind recipe it fails.
> Error: PostCSS plugin postcss-nested requires PostCSS 8.

### Steps to Reproduce
Windows 10
1. Create a fresh project
2. `blitz install tailwind`
3. When I start I get the error.
4. Tried to remove the `.blitz` folder but still the same error.

### Versions

```
You are using alpha software - if you have any problems, please open an issue here:
      https://github.com/blitz-js/blitz/issues/new/choose

debug: blitzPkgPath: F:\Development\example\node_modules\blitz\dist\index.js
debug: cliPkgPath: F:\Development\example\node_modules\@blitzjs\cli\lib\src\index.js

Windows 10 | win32-x64 | Node: v14.2.0

blitz: 0.26.0 (global)
blitz: 0.26.0 (local)

  Package manager: yarn
  System:
    OS: Windows 10 10.0.19041
    CPU: (6) x64 Intel(R) Core(TM) i5-8600K CPU @ 3.60GHz
    Memory: 4.84 GB / 15.93 GB
  Binaries:
    Node: 14.2.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.8 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/cli: 2.12.1 => 2.12.1
    @prisma/client: 2.12.1 => 2.12.1
    blitz: 0.26.0 => 0.26.0
    react: 0.0.0-experimental-4ead6b530 => 0.0.0-experimental-4ead6b530
    react-dom: 0.0.0-experimental-4ead6b530 => 0.0.0-experimental-4ead6b530
    typescript: 4.1.2 => 4.1.2
```

### Other

```
F:\Development\example>npm run start

> example@1.0.0 start F:\Development\example
> blitz start

You are using alpha software - if you have any problems, please open an issue here:
      https://github.com/blitz-js/blitz/issues/new/choose

> ⚠️  Git config core.excludesFile is unset. Inferring .gitignore file locations.
√ Prepped for launch
Loaded env from F:\Development\example\.blitz\caches\dev\.env
warn  - You have enabled experimental feature(s).
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use them at your own risk.

ready - started server on http://localhost:3000
info  - Using external babel configuration from F:\Development\example\.blitz\caches\dev\babel.config.js
error - ./app/styles/index.css (F:/Development/example/node_modules/css-loader/dist/cjs.js??ref--5-oneOf-6-1!F:/Development/example/node_modules/next/dist/compiled/postcss-loader/cjs.js??ref--5-oneOf-6-2!./app/styles/index.css)
Error: PostCSS plugin postcss-nested requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users
```

**Comment by agamm** [comment]
Upon some research it seems like fixing in package.json, `autoprefixer` to be:
```
    "autoprefixer": "^9.0.0",
```

Fixes the problem.

**Comment by flybayer** [comment]
We just fixed it in #1533! Solution is to install `postcss` as a dependency
